akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/Effect.scala

```diff
@@ -230,6 +233,64 @@ object Effect {
 
   final case class TimerCancelled(key: Any) extends Effect
 
+  /**
+   * The behavior initiated an ask of a targeted recipient.
+   */
+  final class AskInitiated[Req, Res, T](
+      val target: RecipientRef[Req],
+      val responseClass: Class[Res],
```

Comment:
I think we should capture the timeout as well, even if probably the same for all asks in an actor in most cases it _can_ be different, so good to allow asserting on that as well.